deployer/scripts/uuid_migrate.sh

```diff
@@ -88,7 +91,7 @@ function run_uuid_migration() {
     [[ "${OPS_PROJECTS[@]}" =~ "$index" ]] && continue
 
     uid=$(oc get project "$index" -o jsonpath='{.metadata.uid}')
-    create_alias $index $uid
+    create_alias ${PROJ_PREFIX}$index $uid
```

Comment:
Per our discussion yesterday, we won't be needing to make any changes for UUID migration -- we will have aliases that match the current index format added to our index template to map the new index formats to.